usvm-ts/src/main/kotlin/org/usvm/util/EtsFieldResolver.kt

```diff
@@ -59,7 +66,12 @@ private fun tryGetSingleField(
     }
     if (classes.size == 1) {
         val clazz = classes.single()
-        return clazz.fields.single { it.name == fieldName }
+        val fields = clazz.fields.filter { it.name == fieldName }
+        when (fields.size) {
+            0 -> error("No field with name '$fieldName' in class '${clazz.name}'")
+            1 -> return fields.single()
+            else -> error("Multiple fields with name '$fieldName' in class '${clazz.name}': $fields")
+        }
```

Comment:
It is similar to the code from 21-27 lines, but looks different. Let's write them in the same style